3-reveal/migrations/5-IRS/4-Zambia-2020/deploy/zambia_focus_area_irs_base.psql

```diff
@@ -0,0 +1,155 @@
+-- Deploy reveal_irs_zambia_2020:zambia_focus_area_irs_base to pg
+-- requires: reveal_transaction_tables:plans
+-- requires: reveal_irs_zambia_2019:zambia_plan_jurisdictions
+-- requires: reveal_irs_zambia_2019:zambia_jurisdictions
+-- requires: reveal_irs_zambia_2019:zambia_irs_structures
+-- requires: utils:count_elements
+-- requires: utils:array_concat_agg
+
+BEGIN;
+
+SET search_path TO :"schema",public;
+
+CREATE MATERIALIZED VIEW IF NOT EXISTS zambia_focus_area_irs_base
```

Comment:
Instead of a new migration for this, could we simply rename the materialized view `zambia_focus_area_irs` and its indices to `zambia_focus_area_irs_base`?